agent/testflinger_agent/client.py

```diff
@@ -365,3 +365,39 @@ def post_influx(self, phase, result=None):
             )
         except InfluxDBClientError as exc:
             logger.error(exc)
+
+    def post_status_update(self, job_queue, webhook, events):
```

Comment:
It's nice to add type annotations when we add new code at least. Can we go ahead and do that here like you did for EventEmitter?